Temporary patch to keep polling for battery status when fully charged

--
--- src/dkp-device-supply.c	2009-07-06 04:09:34.000000000 -0700
+++ src/dkp-device-supply.c	2011-05-06 20:04:59.000000000 -0700
@@ -664,9 +664,9 @@ dkp_device_supply_setup_poll (DkpDevice 
 
 	g_object_get (device, "state", &state, NULL);
 
-	/* if it's fully charged, don't poll at all */
-	if (state == DKP_DEVICE_STATE_FULLY_CHARGED)
-		goto out;
+	/* if it's fully charged, continue to poll
+	 * TODO(bleung): Restore previous behavior when battery status changes are reported.
+	 */
 
 	/* if it's unknown, poll faster than we would normally */
 	if (state == DKP_DEVICE_STATE_UNKNOWN &&
